arm64: dts: marvell: Change EMMC mode used for 7215-A1

Pavan-Nokia opened the pull request:
Change EMMC mode from HS400 to HS200 as HS400 mode is not supported by sdhci-xenon-phy driver

mssonicbld commented:
/azp run

paulmenzel commented on patch/0011-arm64-dts-marvell-Add-Nokia-7215-IXS-A1-board.patch:
```diff
@@ -11,17 +11,17 @@ Signed-off-by: Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
 Tested-by:  Natarajan Subbiramani <natarajan.subbiramani.ext@nokia.com>
```
As it’s not simply copied anymore, please update the commit message description regarding the removal of the eMMC mode HS400.

paulmenzel reviewed (changes requested):
> Change EMMC mode from HS400 to HS200

This sounds strange, as HS200 has been listed in the DTS before. Maybe:

> Remove HS400 as supported eMMC mode

Please amend the commit message, how you verified that HS400 is not supported (schematics?), and how you test that HS200 works as expected.

mssonicbld commented:
/azp run

Pavan-Nokia commented:
@saiarcot895 @paulmenzel
I have addressed the review comments to update the commit as well as the PR details.
Please help merge this PR
Thank you

Pavan-Nokia commented:
@saiarcot895 @paulmenzel
Please help merge this PR
Thank you

Pavan-Nokia commented:
> No idea, if I asked this, but are you going to get this upstream?

Yes, this modification is in the process of being up streamed parallelly

mssonicbld commented:
/azp run
